extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { memo } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { STATE_ERROR } from '../constants';
+
+const TrackError = memo( ( { link } ) => (
```

Comment:
If this is conditionally rendered to indicate a change of state we should consider using `a11y.speak()` to announce the change of state. Or if you use the built-in `Notice` you get that for free!

Reply:
I don't think we have `Notice` on the frontend. We would need to evaluate how much load it adds when we include it

Reply:
@jeryj Perhaps you've considered this from an a11y perspective already? We could/should address in a followup.

Reply:
I've been thinking about this. I'm thinking the "`header`" of the player will be a `aria-live` region. Maybe we should generate an error there as well, not just in the track list? It can be visually hidden as well, if needed. Or maybe the tracklist should be a `aria-live` region as well. I haven't looked into `Notice` or `a11y.speak()` and what those dependencies would entail.

Reply:
I'm making a card with a reference to this thread.